packages/aws-cdk-lib/aws-ec2/lib/instance.ts

```diff
@@ -323,6 +323,24 @@ export interface InstanceProps {
    * @default - no placement group will be used for this instance.
    */
   readonly placementGroup?: IPlacementGroup;
+
+  /**
+   * Whether the instance is enabled for AWS Nitro Enclaves.
+   *
+   * You can't set both `enclaveEnabled` and `hibernationEnabled` to true on the same instance.
+   *
+   * @default - false
+   */
+  readonly enclaveEnabled?: boolean;
+
+  /**
+   * Whether the instance is enabled for hibernation.
+   *
+   * You can't set both `enclaveEnabled` and `hibernationEnabled` to true on the same instance.
+   *
```

Comment:
wdyt about adding doc link here too to give a reference to the user while setting this value?
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance-hibernationoptions.html

Reply:
Thank you. I think it would be better if there was a link, so I added one.

Reply:
Thank you @mazyu36